crates/prql-parser/src/lexer.rs

```diff
@@ -347,29 +347,19 @@ fn escaped_character() -> impl Parser<char, char, Error = Cheap<char>> {
         just('n').to('\n'),
         just('r').to('\r'),
         just('t').to('\t'),
-        (just('u').ignore_then(
+        (just("u{").ignore_then(
```

Comment:
The canonical way would be to use `.delimited_by`, but this is OK given it seems to work and these can't be nested

Reply:
Oh, I didn't know about that one. We could use that one instead if you want, but then we can't use `at_least` and `at_most`. I noticed that the Rust compiler is friendly enough to actually tell you that you can at most use 6 characters if you try to use more than that.

Reply:
I think my message was unclear. We can still do the `at_least` / `at_most`. The `.delimited_by` would be for the brackets. Here's an example of it: https://github.com/PRQL/prql/blob/33b71715136380255482e902c8afe92bd5a8fe10/crates/prql-parser/src/expr.rs#L114